data/global/excel/cubemain.txt

```diff
@@ -957,7 +957,6 @@ Item + Corruption Orb -> If item corrupteddummy = belt sock	1				100	15	362	26
 Item + Corruption Orb -> If item corrupteddummy = belt brick	1				100	15	362	25		2	belt,any	ka3						usetype,rar			100											corrupted			1	1	corruptedDummy			101	101																																																																0
 Item + Corruption Orb -> If item corrupteddummy = 100, apply strongest positive boots start	1				100	15	362	100		2	boot,any	ka3						useitem			100	oskill	100	54	1	1						corrupted			1	1	corruptedDummy			101	101																																																																0
 Item + Corruption Orb -> If item corrupteddummy = 100, apply strongest positive	1				100	15	362	99		2	boot,any	ka3						useitem			100	red-dmg%	100		2	3						corrupted			1	1	corruptedDummy			101	101																																																																0
-Item + Corruption Orb -> If item corrupteddummy = 100, apply strongest positive	1				100	15	362	98		2	boot,any	ka3						useitem			100	nofreeze	100		1	1						corrupted			1	1	corruptedDummy			101	101																																																																0

```

Comment:
Why delete this outcome? Isn't that a strong outcome for folks? Or is that somewhere lower down in the roll range?

Reply:
Talked via discord. Sounds like this was the case.